readme.txt

```diff
@@ -234,6 +234,7 @@ Remember to always make a backup of your database and files before updating!
 
 = [6.5.1] TBD =
 
+* Tweak - Add note to `tribe_create_venue()` docblock to indicate future deprecation.
```

Comment:
```suggestion
* Tweak - Add note to `tribe_create_venue()`, `tribe_create_event()`.... docblock to indicate future deprecation.
```
Maybe we consolidate these changelog entries into one when you're done.

Reply:
I like that.